### PR TITLE
Update README.md on 4x tree

### PR DESCRIPTION
**Pull request by karmicdice**
Update README.md on 4x tree to reflect 4x installation via Composer.

Refers to #724

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,6 +25,12 @@ In case you want to use a custom app dir name (e.g. `/myapp/`):
 composer create-project --prefer-dist cakephp/app myapp
 ```
 
+If you want to install 4.x branch in your custom app directory
+
+```bash
+composer create-project --prefer-dist --stability=dev cakephp/app:4.x myapp
```
Comment (dereuromark) [inline README.md]: Not sure if we should duplicate, instead better modify the existing row maybe?
What do others think?

Comment (karmicdice) [reply README.md]: You're right. This isn't master branch. I am directly requesting this pull on 4x branch. It is implied that the user is looking at 4x branch and wants to use 4x. I will make the change.

Comment (saeideng) [reply README.md]: I'm not sure you need to `--stability=dev` 

it works without this , let `cakephp/app` handles this

**Comment by jdecode** [comment]
Why would `composer create-project --prefer-dist cakephp/app:4.x` be still failing?

Getting `Could not find package cakephp/app with version 4.x` error.

Screenshot from 2019-11-07 23-44-17


Adding the flag `--stability=dev` gets the job done though.

Thanks

**Comment by karmicdice** [comment]
> Adding the flag --stability=dev gets the job done though.

Because 4.x is at dev?

**Comment by jdecode** [comment]
> > Adding the flag --stability=dev gets the job done though.
> 
> Because 4.x is at dev?

Shouldn't the documentation reflect what is available (and should work without having to check if the branch is in dev or GA stage?)

**Comment by karmicdice** [comment]
Hmm. Ok.. See... This is currently pushed to 4.x branch which is dev. When user explicitly comes on github and changes the branch to 4.x- this is the readme they will see. It is implied to the user that what he is looking at is dev. The README at **master** will require telling user not when user knows he is at a different branch already.

**Pull request merged**